exercises/bank.js

```diff
@@ -41,6 +72,12 @@ class Bank{
 			//"account <accountNumber> now has <new account amount>
 	}
 	withdraw( accountNumber, amount ){
+		if (!this.checkForAccount(accountNumber)){
+			return 'account does not exist';
+		} else {
```

Comment:
you're not supposed to empty the account!  You're supposed to refuse to delete the account if it isn't empty!

Reply:
Guess I'll have to add a test for that